build.sbt

```diff
@@ -1422,7 +1423,8 @@ lazy val `engine-common` = project
     Test / javaOptions ++= Seq(
     ),
     libraryDependencies ++= Seq(
-      "org.graalvm.polyglot" % "polyglot" % graalMavenPackagesVersion % "provided"
+      "org.graalvm.polyglot"                   % "polyglot"              % graalMavenPackagesVersion % "provided",
+      "com.github.plokhotnyuk.jsoniter-scala" %% "jsoniter-scala-macros" % "2.28.5"                  % "provided"
```

Comment:
The major reason why I extracted `engine-common` from `polyglot-api` was to avoid dependency on JSON parsing:

- #8894

please don't return it back. E.g. Don't make `engine-common` depend on `"com.github.plokhotnyuk.jsoniter-scala" %% "jsoniter-scala-macros"` - not even `provided`, please.

Reply:
Yes, I realized this was a mistake. That's the problem with naming something `common` :)

Reply:
Shall I rename it to `engine-lightweight-common` - e.g. opposite of [heavyweight](http://wiki.apidesign.org/wiki/HeavyWeight)?